### PR TITLE
[SPARK-17645][MLLIB][ML][FOLLOW-UP] document minor change

### PR DESCRIPTION
**Pull request by mpjlu**
## What changes were proposed in this pull request?
Add FDR test case in ml/feature/ChiSqSelectorSuite.
Improve some comments in the code.
This is a follow-up pr for #15212.

## How was this patch tested?
ut

### CODE REVIEW

File: python/pyspark/ml/feature.py [hunk 1]
```diff
@@ -2629,6 +2629,8 @@ class ChiSqSelector(JavaEstimator, HasFeaturesCol, HasOutputCol, HasLabelCol, Ja
     """
     .. note:: Experimental
 
+    Chi-Squared feature selection, which selects categorical features to use for predicting a
+    categorical label.
     Creates a ChiSquared feature selector.
```
Comment (jkbradley) [inline python/pyspark/ml/feature.py]: This line is now duplicated info, so I'd remove it.

**Review by jkbradley — COMMENTED** [review]
Thanks for the PR!  Responding to your comment in the other PR, I do think it's worth creating a new test set (or replacing the current one?) which lets us test FDR.

**Comment by jkbradley** [comment]
Also, could you please change the PR description to be self-contained (rather than just referencing another PR)?  The description becomes the commit message.

**Comment by mpjlu** [comment]
Hi @jkbradley , I have updated this PR per your comments. Thanks.

**Review by srowen — APPROVED** [review]

File: mllib/src/test/scala/org/apache/spark/ml/feature/ChiSqSelectorSuite.scala [hunk 2]
```diff
@@ -35,22 +35,63 @@ class ChiSqSelectorSuite extends SparkFunSuite with MLlibTestSparkContext
...
-      library(stats)
-      x1 <- c(8.0, 0.0, 0.0, 7.0, 8.0)
-      x2 <- c(7.0, 9.0, 9.0, 9.0, 7.0)
-      x3 <- c(0.0, 6.0, 8.0, 5.0, 3.0)
-      y <- c(0.0, 1.0, 1.0, 2.0, 2.0)
-      chisq.test(x1,y)
-      chisq.test(x2,y)
-      chisq.test(x3,y)
+    /*
+     *  Contingency tables
```
Comment (jkbradley) [inline mllib/src/test/scala/org/apache/spark/ml/feature/ChiSqSelectorSuite.scala]: It's kind of nice seeing these tables, but for validation, it's much easier for users to check using R.  Could you please include a section for verifying the results with R, like there was before?

**Comment by jkbradley** [comment]
Thanks @mpjlu !  The changes look good, except that I'd like to have a code snippet for verifying with R.

**Comment by mpjlu** [comment]
Thanks @jkbradley @srowen , I have added a code snippet for verifying with R.

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**